addons/account/models/account_report.py

```diff
@@ -324,6 +330,8 @@ def _get_copied_code(self):
         :return: an unique code for the copied account.report.line
         '''
         self.ensure_one()
+        if not self.code and self.code != "":
```

Comment:
wait... what?  if self.code is falsy but not an empty string you keep it, but raise en error if it's an empty string?

that's non sensical

* remove the constraint you added
* __get_copied_code_ returns False if not self.code
* if a line has a falsy code, it cannot be referenced

Reply:
Thanks for the review. I have changed the code according to your suggestion which is indeed much better